### PR TITLE
Add public folder for front end code

### PR DESCRIPTION
**Pull request by benrondeau**
@jakepeyser Please approve request if this works for you (but don't delete the branch please).

### CODE REVIEW

**Comment by benrondeau** [comment]
@blumareks This is where the our front end code will live

File: public/README.me [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Front end Code
```
Comment (jakepeyser) [inline public/README.me]: What is this `README.me` file? I've never seen this file format and there is a blank `README.md`, as well

**Comment by benrondeau** [comment]
@jakepeyser Fixed the README error. Please approve if it looks good

**Comment by jakepeyser** [comment]
@benrondeau I made a large update to the controller today, so if you are not actively using this branch I would delete it and make a new one off of the `master` branch so that you get those updates

**Pull request merged**